### DISCUSSION TITLE
Naming of newscast vs. OTA for US network affiliates

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#191] [Q&A] [answered]
For local affiliates of US networks (ABC, CBS, NBC and Fox), a distinction is needed between naming of a local station's newscast stream vs. naming of their regular OTA (over the air) stream. I'm willing to perform a clean up of our playlists, but need consensus on the best approach. 

It seems logical that the OTA stream sticks with our standard naming. For example:
**Fox Atlanta GA (WAGA-TV) (720p)**

What then should be done differently with the newscast stream?  Perhaps insert "News" after the network name:
**Fox News Atlanta GA (WAGA-TV) (720p)**

...or drop the network name and lead with the call sign (some of this approach already exists in us_tubi.m3u):
**WAGA-TV News Atlanta GA (720p)**

...or:
**Fox (newscast) Atlanta GA (WAGA-TV) (720p)**

Thoughts?

**Comment by BellezaEmporium** [comment, marked as answer]
The first option seems to be better for me, because the end user won't know all call signs that are available around the US.

**Reply by freearhey** [reply]
I chose these two options, too.

> Fox Atlanta GA (WAGA-TV) (720p)
> Fox News Atlanta GA (WAGA-TV) (720p)

**Reply by MapGuy11** [reply]
I change it up a little Fox 5 Atlanta GA (WAGA-TV) (720p)
Then FOX 5 Atlanta News (WAGA-TV) (720p)
